Mypy typing validation error: error: "type" has no attribute  with column_formatters_detail

### Checklist

- [x] The bug is reproducible against the latest release or `master`.
- [x] There are no similar issues or pull requests to fix it yet.

### Describe the bug

I have custom formatter for my JSONB field:

```python
    column_formatters_detail = {
        models.Email.payload: lambda m, _: Markup(
            f"""
            <pre>{json.dumps(m.payload, indent=4, ensure_ascii=False)}</pre>
            """
        ),
    }
```

It works well, except I got complains from mypy about the incorrect type when I access attribute of the object (in my case it's m.payload):
> app/admin.py:185: error: "type" has no attribute "payload"  [attr-defined]

<img width="721" alt="Image" src="https://github.com/user-attachments/assets/67b29733-b1a5-44a9-b707-055a909545e5" />

I'm not sure why that first parameter passed as a type `type`?

### Steps to reproduce the bug

_No response_

### Expected behavior

Proper type defined for the first parameter and no mypy errors on validation.

### Actual behavior

Mypy raise an error

### Debugging material

_No response_

### Environment

sqladmin==0.20.1

### Additional context

_No response_

--- a/sqladmin/_types.py
+++ b/sqladmin/_types.py
@@ -35,6 +35,13 @@ ENGINE_TYPE = Engine | AsyncEngine
 MODEL_ATTR = str | InstrumentedAttribute
 SESSION_MAKER = sessionmaker | async_sessionmaker
 
+# Signature of a user supplied column formatter. The first argument is the model
+# *instance* being rendered, the second is the name of the attribute and the
+# optional third one is the current request.
+COLUMN_FORMATTER_TYPE: TypeAlias = (
+    Callable[[Any, Any], Any] | Callable[[Any, Any, Request], Any]
+)
+
 T = TypeVar("T")
 
 

--- a/sqladmin/models.py
+++ b/sqladmin/models.py
@@ -37,6 +37,7 @@ from sqladmin._queries import Query
 from sqladmin._types import (
     _UNSET,
     BASE_FORMATTERS_TYPE,
+    COLUMN_FORMATTER_TYPE,
     MODEL_ATTR,
     SESSION_MAKER,
     ColumnFilter,
@@ -291,7 +292,7 @@ class ModelView(BaseView, metaclass=ModelViewMeta):
         ```
     """
 
-    column_formatters: ClassVar[dict[MODEL_ATTR, Callable[..., Any]]] = {}
+    column_formatters: ClassVar[dict[MODEL_ATTR, COLUMN_FORMATTER_TYPE]] = {}
     """Dictionary of list view column formatters.
     Columns can either be string names or SQLAlchemy columns.
 
@@ -305,9 +306,9 @@ class ModelView(BaseView, metaclass=ModelViewMeta):
     ???+ formatter
         ```python
         def formatter(model, attribute, request):
-            # `model` is model instance
-            # `attribute` is a Union[ColumnProperty, RelationshipProperty]
-            # `request` is a starlette.requests.Request
+            # `model` is the model instance
+            # `attribute` is the name of the attribute being rendered
+            # `request` is a starlette.requests.Request, and is optional
             pass
         ```
     """
@@ -427,7 +428,7 @@ class ModelView(BaseView, metaclass=ModelViewMeta):
         ```
     """
 
-    column_formatters_detail: ClassVar[dict[MODEL_ATTR, Callable[..., Any]]] = {}
+    column_formatters_detail: ClassVar[dict[MODEL_ATTR, COLUMN_FORMATTER_TYPE]] = {}
     """Dictionary of details view column formatters.
     Columns can either be string names or SQLAlchemy columns.
 
@@ -441,9 +442,9 @@ class ModelView(BaseView, metaclass=ModelViewMeta):
     ???+ formatter
         ```python
         def formatter(model, attribute, request):
-            # `model` is model instance
-            # `attribute` is a Union[ColumnProperty, RelationshipProperty]
-            # `request` is a starlette.requests.Request
+            # `model` is the model instance
+            # `attribute` is the name of the attribute being rendered
+            # `request` is a starlette.requests.Request, and is optional
             pass
         ```
     """

